### ISSUE TITLE
Read Me says 'cucumber formatting' but doesn't explain

### ISSUE THREAD

**Issue by karlhigley** [tapangi/rgmanager#10]
The README says:

"Please use cucumber formatting for issues if possible. You can also include a skill level for the issue if you like (easy, intermediate, advanced)"

An explanation of what cucumber formatting looks like would be really helpful.

**Comment by psteininger** [comment]
Labeling issues as easy, medium and hard is a subjective process. What's easy for me may be hard for you. what appears simple for you may in fact be quite complex under the hood. 

What I would suggest is being precise and verbose in the issues. This will allow everyone to gain a better understanding of symptoms and come up with the right solutions and implement them.

In regards to RailsGirls, the key thing is to keep adding features as outlined in the README. Emily did an outstanding job outlining the goals and scope. The focus should be features, wireframes, and implementations of these.

**Comment by karlhigley** [comment]
Agreed on skill levels, and on focusing on features.  Still wondering what is meant by 'cucumber formatting.'

**Comment by districtem** [comment]
There was some talk to add issues in "cucumber formatting" at the last meetup.  I'm searching around the web for easy cucumber tutorials.  At the meetup we thought Cucumber would allow for readable feature/issue notification I found this tutorial...what do you all think to add the link to the README? http://cuke4ninja.com/sec_cucumber_jargon.html
